### ISSUE TITLE
go get convey is failing; undefined: errors.ls

### ISSUE THREAD

**Issue by FahimSiddiqui** [smartystreets/goconvey#600]
Hi Team,

I am facing an issue while getting convey package.

**Error:**
Command: `go get github.com/smartystreets/goconvey/convey`
 **# github.com/smartystreets/assertions
 go\src\github.com\smartystreets\assertions\type.go:146:6: undefined: errors.Is**

I have been using Convey since a long time, this issue is happening since a last few days after I reinstalled all the packages.

Go Version:
**go version go1.12beta2 windows/amd64  - (Windows 10)** 

I have tried to install `go get github.com/smartystreets/assertions` as well but that did not help.

Can someone please check.

Thanks.

**Comment by sgnn7** [comment]
I've hit this too today as well. This is because goconvey is depending on [smartystreets/assertions](https://github.com/smartystreets/assertions) repo and they recently [made use of new APIs](https://github.com/smartystreets/assertions/commit/b63c46aebc02f130a052a785f4f27bc938f2a4f3#diff-65fa0d5b4af4e755a507061fdfa76ddbR146) that are only available in Golang v1.13. 

tl;dr: If you use `goconvey` I think you just have to use v1.13 of Golang now (or use the v1.0.1 of the `smartystreets/assertions` dependency).

**Comment by FahimSiddiqui** [comment]
Thanks @sgnn7 . I have upgraded to GO v1.14 . It worked perfectly fine after that.
